### DISCUSSION TITLE
How many QPS does ADG have?

### DISCUSSION THREAD

**Discussion by ghost** [AdguardTeam/AdGuardHome#3809] [General]
I'm a little curious about the query rate (QPS) of ADG. Has anyone tested it? What is the peak number of requests processed per second? How does it compare to [bind](https://www.isc.org/bind/)?

**Comment by ghost** [comment]
@ameshkov @EugeneOne1 @ainar-g
Has the official done relevant tests?

**Reply by ameshkov** [reply]
We never tested this. Note, that AdGuard Home is intended for personal use so this kind of testing is not too relevant to what we do.

**Reply by ainar-g** [reply]
To add to Andrey's answer, the actual numbers will vary wildly between machines, networks, upstreams, and configurations.
